code/go/0chain.net/blobbercore/allocation/attributesfilechange.go

```diff
@@ -27,7 +27,8 @@ func (ac *AttributesChange) ProcessChange(ctx context.Context, _ *AllocationChan
 	path = filepath.Clean(path)
 
 	// root reference
-	ref, err = reference.GetReferencePath(ctx, ac.AllocationID, ac.Path)
+	ref, err = reference.GetReferencePath2(ctx, ac.AllocationID, ac.Path)
```

Comment:
Name of function seems wrong

Reply:
This is a new function in code/go/0chain.net/blobbercore/reference/referencepath.go in line 25.

Reply:
This is fine by me